File: C/Suma_de_dos_enteros.c
//Programa de suma
#include <stdio.h>

int main() {
    int entero1; /* Primer numero entero a introducir */
    int entero2; /* Segundo numero entero a introducir */
    int suma; /*Variable que se va almacenar el resultado */

    printf("Introduzca el primer numero entero:\n"); /* Indicador */
    scanf("%d", &entero1); /* Lee un numero entero */

    printf("Introduzca el segundo numero entero:\n"); /* Indicador */
    scanf("%d", &entero2); /* Lee el segundo numero entero */

    suma = entero1 + entero2; /* Hace la operacion */

    printf("La suma de %d y de %d es de: %d", entero1, entero2, suma); /* Imprime suma */
    
    return 0;
} /* Fin de la funcion */

File: Kotlin/Suma_de_dos_enteros.kt
//Suma de dos enteros
//Se inicia la funcion main
fun main() {
    var entero1: Int
    var entero2: Int
    var suma: Int

    println("Ingrese el primer numero entero:")
    entero1 = readLine()!!.toInt() //Aqui se lee el numero "entero1" y el "!!" es para que no sea nulo, y el "toInt()" es para convertirlo a entero porque readLine() lo devuelve como String

    println("Ingrese el segundo numero entero:")
    entero2 = readLine()!!.toInt()

    suma = entero1 + entero2 // Aqui se hace la operacion que y se guarda en el "var suma"

    println("La suma de $entero1 mas $entero2 es igual a: $suma")
} 
//Fin de la funcion main